### PR TITLE
Prepare v1.8.2 release

### PR DESCRIPTION
**Pull request by esigo**

### CODE REVIEW

**Review by marcalff — COMMENTED** [review]
Please fix this in new file DEPRECATED.md as well:

Version: TO-BE-RELEASED-VERSION
Date: TO-BE-RELEASED-DATE

**Comment by esigo** [comment]
> Please fix this in new file DEPRECATED.md as well:
> 
> Version: TO-BE-RELEASED-VERSION Date: TO-BE-RELEASED-DATE

@marcalff thanks, done!

**Review by marcalff — APPROVED** [review]
Thanks. LGTM

File: DEPRECATED.md [hunk 1]
```diff
@@ -60,8 +60,8 @@ N/A
 
 #### Announcement (Jaeger)
 
-* Version: TO-BE-RELEASED-VERSION
-* Date: TO-BE-RELEASED-DATE
+* Version: 1.8.2
+* Date: 2023-01-31
 * PR: [DEPRECATION] Deprecate the Jaeger exporter
   [#9999](https://github.com/open-telemetry/opentelemetry-cpp/pull/9999)
```
Comment (ThomsonTan) [inline DEPRECATED.md:66]: Should the below PR URL be updated as well?

**Review by ThomsonTan — APPROVED** [review]
Thanks!

File: DEPRECATED.md (hunk 1 above)
Comment (esigo) [reply DEPRECATED.md]: Thanks, done!

**Review by lalitb — APPROVED** [review]

**Pull request merged**